src/odb/include/odb/dbTypes.h

```diff
@@ -207,22 +208,22 @@ class dbGroupType
   /// The explicit orientation must be a string of one of the
   ///  following values: "PHYSICAL_CLUSTER", "VOLTAGE_DOMAIN".
   ///
-  dbGroupType(const char* type);
+  dbGroupType(const char* orient);
```

Comment:
How come this changed from type to orient?

Reply:
I came across it while editing. The name in the *.cpp file for this is "orient", so adpted them to be the same.
(clang tidy warning about inconsistent names in declaration and implementation)

Reply:
Agreed - orient is wrong.